notebooks/minicpm-o-omnimodal-chatbot/minicpm-o-omnimodal-chatbot.ipynb

```diff
@@ -0,0 +1,808 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #46.    generate_audio = False
</pre><p>as audio generation is not supported, I think it is better to remove these flags from notebook for now</p><p> </p><p>Also not sure if onednn logs should be here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2851/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
Sure I will remove

Reply:
done